pkg/proc/errors.go

```diff
@@ -0,0 +1,37 @@
+package proc
```

Comment:
I'm not a fan of having a file specifically to declare a series of miscellaneous error. ErrNoRuntimeAllG should be where it is returned (goroutine_cache.go). The rest should either stay at their original location or maybe go in target.go (they are returned by the backends).